itunes.py

```diff
@@ -23,16 +30,21 @@ def unset_track(self, view):
 
     def _format_status_string(self):
         track_data = self.get_track_data()
-        if not track_data: return
+        if not track_data:
+            return
         return "▶ {artist} - {name}".format(**track_data)
 
     def get_track_data(self):
-        track_p = subprocess.Popen("osascript -l JavaScript -e 'Application(\"System Events\").applicationProcesses[\"iTunes\"](); Application(\"iTunes\").currentTrack.name();'", shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
+        track_script = sublime.load_resource("Packages/iTunes/scripts/track.scpt")
+        artist_script = sublime.load_resource("Packages/iTunes/scripts/artist.scpt")
+        track_p = subprocess.Popen("osascript -l JavaScript -e " + shlex.quote(track_script), shell=True, stdout=subprocess.PIPE, stderr=subprocess.PIPE)
```

Comment:
it's a very good way.
but package does not have `.no-sublime-package` file, so this package installed as zipped app.
`sublime.load_resource` works correctly in zipped ?